data/assets/scene/digitaluniverse/stars.asset

```diff
@@ -29,11 +29,51 @@ local stars = {
         Type = "RenderableStars",
         File = speck .. "/stars.speck",
         Texture = textures .. "/halo.png",
-        ColorMap = colorLUT .. "/colorbv.cmap"
+        ColorMap = colorLUT .. "/colorbv.cmap",
+        MagnitudeExponent = 6.2,
+        SizeComposition = "Distance Modulus",
+        RenderMethod = "Texture Based" -- or PSF
     },
     GUI = {
         Path = "/Milky Way/Stars"
     }
 }
-
+--[[
```

Comment:
We should not commit the test grid code to keep it simple.  Better to outsource it into a different asset file that lives in the 'example' folder, for example